bson.cpp

```diff
@@ -424,6 +428,91 @@ void VariantToBsonConverter::_convertSerializable(bson_t *bson, const char *key,
 }
 /* }}} */
 
+/* {{{ MongoDriver\Driver\ReadConcern */
+void VariantToBsonConverter::_convertReadConcern(bson_t *bson, const char *key, Object v)
+{
+	MongoDBDriverReadConcernData* data = Native::data<MongoDBDriverReadConcernData>(v);
+	const char *rp_string = mongoc_read_concern_get_level(data->m_read_concern);
+	bson_t child;
+
+	bson_append_document_begin(bson, key, -1, &child);
+
+	bson_append_utf8(&child, "level", -1, rp_string, -1);
```

Comment:
If you're going to compute string lengths for both fields, you may as well use the `BSON_APPEND_UTF8()` macro. That will have the added effect of translating to `strlen("level")` before the function is called, which may be optimized by the compiler.